juju/js-libjuju
# WD-694 - Introduce generic facades

## Description

Add generic facades where the library will a named facade with the most recent supported version by the controller.

## QA

This is a double QA for this PR and the one over juju-dashboard.

- `yarn build`
- `yarn link`
- Try with the juju-dashboard ([this PR](https://github.com/canonical/jaas-dashboard/pull/1357)): `yarn link @canonical/jujulib`
- Make sure that the dashboard is working properly without errors in the console
